app/models/rook.rb

```diff
@@ -1,5 +1,6 @@
 class Rook < Piece
-  def move
+  def valid_move?(x, y)
```

Comment:
Should put a call to is_obstructed? in here. at least according to the Trello story.